pkg/cli/debug.go

```diff
@@ -646,24 +628,6 @@ func runDebugGCCmd(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-var debugRocksDBCmd = &cobra.Command{
```

Comment:
I assume going forward the easiest way to use these subcommands (including the one I linked elsewhere for versionupgrade) is to use a 20.2 binary?

Reply:
Most of these subcommands are implemented (in a better way) under `cockroach debug pebble`. The few exceptions that don't exist are `checkpoint`, `put`, etc - and so I've just skipped the roachtests that use those until we've figured out an alternative way, or reimplemented them under pebble too. The underlying functionality exists in Pebble already; it's just not exposed to the command line.